pkg/cmd/utils_test.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+	v1 "k8s.io/api/core/v1"
+)
+
+func TestGetVersionFromImage(t *testing.T) {
+	tests := []struct {
+		name       string
+		key        string
+		containers []v1.Container
+		expected   string
+	} {
+		{
```

Comment:
can we have one more testcase, e.g., a/b/c/cloud-provider-vsphere/csi/release/driver:v1.0.1?

Reply:
Added one more test case with multiple level registry name.